dockergen/nginx.tmpl

```diff
@@ -71,6 +72,7 @@ server {
 {{ range $url, $containers := groupByMulti $containersByHost "Env.VIRTUAL_URL" "," }}
     location {{ $url }} {
         proxy_pass http://{{ $host }}{{ replace $url "/" "-" -1 }}u/;
+        proxy_redirect https://{{ $host }}{{ replace $url "/" "-" -1 }}u/ /;
```

Comment:
We have not updated it yet. But you can maybe first go manually into the image (`docker exec`) and install a newer version of nginx from some newer package repository for Ubuntu or some other repository. Let's see if that fixes it. Then we will have more motivation to update other stuff.